src/site/content/en/media/video-and-source-tags/index.md

```diff
@@ -0,0 +1,336 @@
+    viewport.</figcaption>
+  </figure>
+  <figure class="w-figure">
+    <img src="./chrome-android-landscape-video-unstyled.png" alt="Android Chrome screenshot, landscape: unstyled video element overflows
+    viewport.">
+    <figcaption class="w-figcaption">Android Chrome screenshot, landscape: unstyled video element overflows
+    viewport.</figcaption>
+  </figure>
+</div>
+
+You can control video dimensions using JavaScript or CSS. JavaScript libraries
+and plugins such as [FitVids](http://fitvidsjs.com/) make it possible to
+maintain appropriate size and aspect ratio, even for videos from YouTube and
+other sources.
+
+Use [CSS media
+queries](/web/fundamentals/design-and-ux/responsive/#css-media-queries) to
+specify the size of elements depending on the viewport dimensions; `max- width:
+100%` is your friend.
+
+For media content in iframes (such as YouTube videos), try a responsive approach
+(like the one [proposed by John
+Surdakowski](http://avexdesigns.com/responsive-youtube-embed/)).
+
+{% Aside 'caution' %}
+Don't force element sizing that results in an aspect ratio different from the
+original video. Squashed or stretched looks bad.
+{% endAside %}
+
+**CSS:**
+
+```css
+.video-container {
+    position: relative;
+    padding-bottom: 56.25%;
+    padding-top: 0;
+    height: 0;
+    overflow: hidden;
+}
+
+.video-container iframe,
+.video-container object,
+.video-container embed {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+}
+```
+
+**HTML:**
+
+```html
+<div class="video-container">
+  <iframe src="//www.youtube.com/embed/l-BA9Ee2XuM"
+          frameborder="0" width="560" height="315">
+  </iframe>
+</div>
+```
+
+[Try it](https://googlesamples.github.io/web-fundamentals/fundamentals/media/responsive_embed.html)
+
+Compare the [responsive sample](https://googlesamples.github.io/web-fundamentals/fundamentals/media/responsive_embed.html)
+to the [unresponsive version](https://googlesamples.github.io/web-fundamentals/fundamentals/design-and-ux/responsive/unyt.html).
+
+### Device orientation
+
+Device orientation isn't an issue for desktop monitors or laptops, but it's
+hugely important when considering web page design for mobile devices and
+tablets.
+
+Safari on iPhone do a good job of switching between portrait and landscape
+orientation:
+
+<div class="w-columns">
+<figure class="w-figure">
+  <img src="./iphone-video-playing-portrait.png" alt="Screenshot of video playing in Safari on iPhone, portrait.">
+  <figcaption class="w-figcaption">Screenshot of video playing in Safari on iPhone, portrait.</figcaption>
+</figure>
+<figure class="w-figure">
+  <img src="./iphone-video-playing-landscape.png" alt="Screenshot of video playing in Safari on iPhone, landscape.">
+  <figcaption class="w-figcaption">Screenshot of video playing in Safari on iPhone, landscape.</figcaption>
+</figure>
+</div>
+
+Device orientation on an iPad and Chrome on Android can be problematic.
```

Comment:
Is this still true?

Reply:
I believe so. I just looked at a video on iPad safari that does not have `width: 100%` or `max-width: 100%`. It looked roughly like the image below. I could be missing something. I'll investigate further.